python-black.el

```diff
@@ -119,6 +119,22 @@ DISPLAY-ERRORS is non-nil, shows a buffer if the formatting fails."
   (and (= (point-min) beg)
        (= (point-max) end)))
 
+(defun python-black-block ()
+  "Reformats the current org-mode source block."
+  (interactive)
+  (save-mark-and-excursion
+    (when (org-in-block-p '("src" "example"))
+      (let* ((element (org-element-at-point))
+             (src-block-not-empty
+              (string-match
+               "[^\\s\\n]+"
+               (org-element-property :value element)))
+             (area (org-src--contents-area element)))
+        (when src-block-not-empty
```

Comment:
seems like this nested `when` + `let*` + `when` can be one `when-let*` instead?

Reply:
superseeded, I'll use unless in another version.